src/internal/widgets/index.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
```

Comment:
To make widgetization simpler, extracted this shared helper. One function for normal components, one for forward-ref ones